Cargo.lock

```diff
@@ -85,6 +85,51 @@ version = "1.1.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "d468802bab17cbc0cc575e9b053f41e72aa36bfa6b7f55e3529ffa43161b97fa"
 
+[[package]]
+name = "axum"
```

Comment:
Why do we need an axum dependency?

Reply:
no idea, i'll try to figure out where that came from...

Reply:
apparently the `axum` dep is via `tonic`:
<details>

<summary>`cargo tree` output</summary>

```
:; cargo tree -p axum -i
axum v0.5.17
└── tonic v0.8.1
    ├── linkerd-app v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/app)
    │   ├── linkerd-app-integration v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/app/integration)
    │   └── linkerd2-proxy v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd2-proxy)
    ├── linkerd-app-core v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/app/core)
    │   ├── linkerd-app v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/app) (*)
    │   ├── linkerd-app-admin v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/app/admin)
    │   │   └── linkerd-app v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/app) (*)
    │   │   [dev-dependencies]
    │   │   └── linkerd-app-integration v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/app/integration)
    │   ├── linkerd-app-gateway v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/app/gateway)
    │   │   └── linkerd-app v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/app) (*)
    │   ├── linkerd-app-inbound v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/app/inbound)
    │   │   ├── linkerd-app v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/app) (*)
    │   │   ├── linkerd-app-admin v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/app/admin) (*)
    │   │   └── linkerd-app-gateway v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/app/gateway) (*)
    │   ├── linkerd-app-integration v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/app/integration)
    │   ├── linkerd-app-outbound v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/app/outbound)
    │   │   ├── linkerd-app v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/app) (*)
    │   │   └── linkerd-app-gateway v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/app/gateway) (*)
    │   └── linkerd-app-test v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/app/test)
    │       └── linkerd-app-integration v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/app/integration)
    │       [dev-dependencies]
    │       ├── linkerd-app-gateway v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/app/gateway) (*)
    │       ├── linkerd-app-inbound v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/app/inbound) (*)
    │       └── linkerd-app-outbound v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/app/outbound) (*)
    ├── linkerd-app-inbound v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/app/inbound) (*)
    ├── linkerd-app-integration v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/app/integration)
    ├── linkerd-app-outbound v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/app/outbound) (*)
    ├── linkerd-client-policy v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/client-policy)
    │   └── linkerd-app-outbound v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/app/outbound) (*)
    ├── linkerd-opencensus v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/opencensus)
    │   ├── linkerd-app v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/app) (*)
    │   └── linkerd-app-core v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/app/core) (*)
    ├── linkerd-proxy-api-resolve v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/proxy/api-resolve)
    │   ├── linkerd-app-core v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/app/core) (*)
    │   └── linkerd-service-profiles v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/service-profiles)
    │       └── linkerd-app-core v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/app/core) (*)
    ├── linkerd-proxy-identity-client v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/proxy/identity-client)
    │   └── linkerd-app-core v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/app/core) (*)
    ├── linkerd-proxy-tap v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/proxy/tap)
    │   └── linkerd-app-core v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/app/core) (*)
    ├── linkerd-service-profiles v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/service-profiles) (*)
    ├── linkerd-tonic-watch v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/tonic-watch)
    │   ├── linkerd-app-inbound v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/app/inbound) (*)
    │   ├── linkerd-app-outbound v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/app/outbound) (*)
    │   └── linkerd-service-profiles v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/service-profiles) (*)
    ├── linkerd2-proxy-api v0.7.0 (https://github.com/linkerd/linkerd2-proxy-api?branch=alex/outbound#9cfb93f3)
    │   ├── linkerd-app-inbound v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/app/inbound) (*)
    │   ├── linkerd-app-integration v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/app/integration)
    │   ├── linkerd-app-outbound v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/app/outbound) (*)
    │   ├── linkerd-client-policy v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/client-policy) (*)
    │   ├── linkerd-http-route v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/http-route)
    │   │   ├── linkerd-client-policy v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/client-policy) (*)
    │   │   └── linkerd-server-policy v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/server-policy)
    │   │       ├── linkerd-app-core v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/app/core) (*)
    │   │       └── linkerd-app-inbound v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/app/inbound) (*)
    │   ├── linkerd-policy-core v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/policy-core)
    │   │   ├── linkerd-client-policy v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/client-policy) (*)
    │   │   └── linkerd-server-policy v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/server-policy) (*)
    │   ├── linkerd-proxy-api-resolve v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/proxy/api-resolve) (*)
    │   ├── linkerd-proxy-identity-client v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/proxy/identity-client) (*)
    │   ├── linkerd-proxy-tap v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/proxy/tap) (*)
    │   ├── linkerd-server-policy v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/server-policy) (*)
    │   └── linkerd-service-profiles v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/service-profiles) (*)
    │   [dev-dependencies]
    │   ├── linkerd-proxy-tap v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/proxy/tap) (*)
    │   └── linkerd-service-profiles v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/service-profiles) (*)
    └── opencensus-proto v0.1.0 (/home/eliza/Code/linkerd2-proxy/opencensus-proto)
        └── linkerd-opencensus v0.1.0 (/home/eliza/Code/linkerd2-proxy/linkerd/opencensus) (*)
```

</details>

which is interesting, because i _thought_ all of our `tonic` deps disabling defautl features would avoid this, and this PR doesn't add any new `tonic` deps that don't have `default-features = false` AFAICT. hmm.